pkg/executor/core/executor.go

```diff
@@ -229,13 +233,17 @@ func (executor Executor) verifyReferenceForRegoPolicy(ctx context.Context, subje
 			verifierStartTime := time.Now()
 			verifierResult, err := verifier.Verify(errCtx, subjectRef, referenceDesc, referrerStore)
 			if err != nil {
+				verifierErr := errors.ErrorCodeVerifyReferenceFailure.NewError(errors.Verifier, verifier.Name(), errors.EmptyLink, err, nil, errors.HideStackTrace)
 				verifierReport = vt.VerifierResult{
 					IsSuccess:    false,
-					Name:         verifier.Name(), // Deprecating Name in next release, reference to VerifierName instead.
-					Type:         verifier.Type(), // Deprecating Type in next release, reference to VerifierType instead.
+					Name:         verifier.Name(), // Deprecating Name in v2, switch to VerifierName instead.
```

Comment:
Clarification @binbin-li.  Why don't we deprecate this now, and to be removed in V2?

Reply:
It's just for backward compatibility. I keep it in case users have existing Rego query referencing Name and Type.

Reply:
Created issue to track the deprecation: https://github.com/ratify-project/ratify/issues/1707